bigtable-dataflow-parent/bigtable-hbase-dataflow/src/main/java/com/google/cloud/bigtable/dataflow/CloudBigtableScanConfiguration.java

```diff
@@ -18,14 +18,14 @@
 
 import org.apache.hadoop.hbase.client.Scan;
 
-import com.google.bigtable.repackaged.com.google.cloud.grpc.BigtableInstanceName;
-import com.google.bigtable.repackaged.com.google.cloud.hbase.adapters.Adapters;
-import com.google.bigtable.repackaged.com.google.cloud.hbase.adapters.read.DefaultReadHooks;
-import com.google.bigtable.repackaged.com.google.cloud.hbase.adapters.read.ReadHooks;
-import com.google.bigtable.repackaged.com.google.cloud.util.ByteStringer;
-import com.google.bigtable.repackaged.com.google.com.google.bigtable.v2.ReadRowsRequest;
-import com.google.bigtable.repackaged.com.google.com.google.bigtable.v2.RowRange;
-import com.google.bigtable.repackaged.com.google.com.google.bigtable.v2.RowSet;
+import com.google.bigtable.repackaged.com.google.cloud.bigtable.grpc.BigtableInstanceName;
+import com.google.bigtable.repackaged.com.google.cloud.bigtable.hbase.adapters.Adapters;
+import com.google.bigtable.repackaged.com.google.cloud.bigtable.hbase.adapters.read.DefaultReadHooks;
+import com.google.bigtable.repackaged.com.google.cloud.bigtable.hbase.adapters.read.ReadHooks;
```

Comment:
I'm going to fix that.  I'd like to remove the dataflow shaded package, and repackage the protos in bigtable-hbase-shaded.